system/HTTP/RedirectException.php

```diff
@@ -0,0 +1,3 @@
+<?php namespace CodeIgniter\HTTP;
+
+class RedirectException extends \Exception {}
```

Comment:
We already have a RedirectException defined with the Router. I'm tempted to say that we should move the Debug/CustomExceptions file to the `system` directory and make it a catch all for any exceptions used by the framework. 

The only problem with that is making it simple to distribute each of the main elements as usable packages in other applications.